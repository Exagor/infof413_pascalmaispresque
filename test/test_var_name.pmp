begin
    read(10alex)
end