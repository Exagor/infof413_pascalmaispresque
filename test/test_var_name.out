token: begin	lexical unit: BEG
token: read	lexical unit: READ
token: (	lexical unit: LPAREN
token: 10	lexical unit: NUMBER
token: alex	lexical unit: VARNAME
token: )	lexical unit: RPAREN
token: end	lexical unit: END
Variables :
alex   2
